neuralmonkey/vocabulary.py

```diff
@@ -43,29 +43,10 @@ def _is_special_token(word: str) -> bool:
             or word == UNK_TOKEN)
 
 
-def from_file(path: str) -> 'Vocabulary':
-    """Loads vocabulary from a pickled file
-
-    Arguments:
-        path: The path to the pickle file
-
-    Returns:
-        The newly created vocabulary.
-    """
-    if not os.path.exists(path):
-        raise Exception("Vocabulary file does not exist: {}".format(path))
-
-    with open(path, 'rb') as f_pickle:
-        vocabulary = pickle.load(f_pickle)
-    assert isinstance(vocabulary, Vocabulary)
-
-    log("Pickled vocabulary loaded. Size: {} words".format(len(vocabulary)))
-    vocabulary.log_sample()
-    return vocabulary
-
-
 def from_wordlist(path: str, encoding: str = "utf-8") -> 'Vocabulary':
-    """Loads vocabulary from a wordlist.
+    """Loads vocabulary from a wordlist. The file can contain either list of
+    words with no header. Or it can contain words and their counts separated
+    by tab and a header on the first line.
```

Comment:
Jestli to má header a frequencies bych radši místo automatické detekce, tak bych to tady dal jako parametry třeba: `skip_header` a `contains_frequencies`. Uživatel nemusí tušit, jaký formát používá TensorBoard.